armi/reactor/blueprints/tests/test_customIsotopics.py

```diff
@@ -216,18 +220,23 @@ def test_massFractionsAreApplied(self):
             :id: T_ARMI_MAT_USER_INPUT3
             :tests: R_ARMI_MAT_USER_INPUT
         """
-        fuel0 = self.a[0].getComponent(Flags.FUEL)
         fuel1 = self.a[1].getComponent(Flags.FUEL)
         fuel2 = self.a[2].getComponent(Flags.FUEL)
+        fuel6 = self.a[6].getComponent(Flags.FUEL)
         self.assertEqual(self.numCustomNuclides, len(fuel1.p.numberDensities))
         self.assertAlmostEqual(19.1, fuel1.density())
 
-        # density only works with a Custom material type.
-        self.assertAlmostEqual(fuel0.density(), fuel2.density())
         self.assertEqual(
             set(fuel2.p.numberDensities.keys()), set(fuel1.p.numberDensities.keys())
         )  # keys are same
 
+        self.assertAlmostEqual(19.1, fuel2.density())
+        # original material density should not be changed after setting a custom density component
+        self.assertNotAlmostEqual(fuel2.density(), fuel6.density(), places=2)
```

Comment:
change the fuel2 density to something besides 19.1 so that we can confirm it actually happened. these tests seem to indicate the baseline density is 19.1, so the test hasn't confirmed a change

Reply:
The density of 19.1 is what is set in the blueprints above. Fuel blocks 0, 2, and 6 are all using the non-custom material. Fuel blocks 0 and 6 do not have a custom density set, while fuel block 2 has a custom density. Line 235 verifies that the custom density of fuel block 2 is not equal to fuel block 6, therefore the density of fuel block 2 has been changed from the baseline material density.

This test originally tested that fuel block 2 did _not_ change density because the material was not set to custom. Let me know if additional clarification is needed.

Reply:
ooooOOOOoo. its really easy for my brain to skip the  word "not"

could you please assert it is equal to whatever its equal to so its more black and white what the actual density is?